src/util/FactorUtil.js

```diff
@@ -161,7 +167,11 @@ function (Okta, TimeUtil) {
       return 'QUESTION';
     }
     if (provider === 'FIDO' && factorType === 'webauthn') {
-      if (this.settings.get('features.webauthn')) {
+      var hasWebAuthnSupport = navigator && navigator.credentials && navigator.credentials.create;
```

Comment:
can we use `webauth.isNewApiAvailable`?

Reply:
Yea we could.. I would have to import util/webauthn.js so I thought I would do this instead. I'll make changes